src/box/memtx_engine.c

```diff
@@ -1064,6 +1072,9 @@ memtx_engine_new(const char *snap_dirname, bool force_recovery,
 	xdir_create(&memtx->snap_dir, snap_dirname, SNAP, &INSTANCE_UUID,
 		    &xlog_opts_default);
 	memtx->snap_dir.force_recovery = force_recovery;
+	/* This options is useful, only if force_recovery is true */
+	memtx->force_broken_snapshot_recovery =
+		force_recovery ? force_broken_snapshot_recovery : false;
```

Comment:
So the option will be silently ignored if force_recovery is not set? Will not better to give an explicit error for incorrect parameters?

Reply:
Ok, i add message about error in options